cmd/kots/cli/pull.go

```diff
@@ -32,18 +32,19 @@ func PullCmd() *cobra.Command {
 			// strip it if included or else the rewrite images will fail
 
 			pullOptions := pull.PullOptions{
-				HelmRepoURI:         v.GetString("repo"),
-				RootDir:             ExpandDir(v.GetString("rootdir")),
-				Namespace:           v.GetString("namespace"),
-				Downstreams:         v.GetStringSlice("downstream"),
-				LocalPath:           ExpandDir(v.GetString("local-path")),
-				LicenseFile:         ExpandDir(v.GetString("license-file")),
-				ExcludeKotsKinds:    v.GetBool("exclude-kots-kinds"),
-				ExcludeAdminConsole: v.GetBool("exclude-admin-console"),
-				SharedPassword:      v.GetString("shared-password"),
-				CreateAppDir:        true,
-				HelmOptions:         v.GetStringSlice("set"),
-				RewriteImages:       v.GetBool("rewrite-images"),
+				HelmRepoURI:             v.GetString("repo"),
+				RootDir:                 ExpandDir(v.GetString("rootdir")),
+				Namespace:               v.GetString("namespace"),
+				Downstreams:             v.GetStringSlice("downstream"),
+				LocalPath:               ExpandDir(v.GetString("local-path")),
+				LicenseFile:             ExpandDir(v.GetString("license-file")),
+				ExcludeKotsKinds:        v.GetBool("exclude-kots-kinds"),
+				ExcludeAdminConsole:     v.GetBool("exclude-admin-console"),
+				SharedPassword:          v.GetString("shared-password"),
+				CreateAppDir:            true,
+				HelmOptions:             v.GetStringSlice("set"),
+				RewriteImages:           v.GetBool("rewrite-images"),
+				AdminConsoleHostNetwork: v.GetBool("admin-console-host-network"),
```

Comment:
why isn't this also called `UseHostNetwork` like in `kotsadmtypes.DeployOptions`?